projects/dapr/cert-manager-graduation.md

```diff
@@ -0,0 +1,501 @@
+  
+  A gap in documentation is for the SRE / Operations administrator persona in an easily accessible location within the documentation. Tracking issues [4383](https://github.com/dapr/docs/issues/4383) and [4384](https://github.com/dapr/docs/issues/4384) have been created in the Dapr docs repo to track recommendations to remediate the gaps.
+
+- [x] **Document the project's release process and guidelines publicly in a RELEASES.md or equivalent file that defines:** 
+
+  - [x] Release expectations (scheduled or based on feature implementation)
+  
+  The project documents their release process in a [Release Process document](https://github.com/dapr/community/blob/master/release-process.md) that is linked to the [RELEASE.md](https://github.com/dapr/dapr/blob/master/RELEASE.md) file. Releases are aimed to be scheduled on a quarterly basis, however, the release cadence is more frequent with a healthy number of features, contributions and quick security fixes.
+  
+  Each release has a tracking issue that is tracking milestones per component to be added to the current release. Minor release example: [Dapr v1.10.09 Security release](https://github.com/dapr/dapr/issues/6685)
+  
+  Major releases also have a tracking board to provide further information and management for the release. Example: [Dapr v1.5.0 Tracking Issue](https://github.com/dapr/dapr/issues/8017) and [Dapr v1.5.0 Tracking Board]
+  
+  There is a discrepancy between the Support Release Policy and the Release Process - [Issue #8188 in the Dapr repo](https://github.com/dapr/dapr/issues/8188) has been created to track clarifications to each. 
+  
+  - [ ] Tagging as stable, unstable, and security related releases
+
+  Releases aren't specifically tagged as stable, unstable or security. They are [tagged](https://github.com/dapr/dapr/tags) as either a stable release or a "Release Candidate", which is inherently unstable as it is still undergoing testing prior to stable release. 
+  
+  However, a tracking issue may be specified as a security release in the tracking issue for the Minor release, but not tagged as such.
+  
+  Bug fixes are included in both Release Candidates and Stable releases. They can be included in a major or minor release alongside other features, or solely in a minor release and are documented thoroughly. Example: [Dapr 1.14.4 release](https://github.com/dapr/dapr/releases/tag/v1.14.4).
+  
+  [Issue #8188 in the Dapr repo](https://github.com/dapr/dapr/issues/8188) has been created to track tagging policy clarification.
+  
+  - [ ] Information on branch and tag strategies
+
+  The Dapr project provides information on branch strategies in the [Maintainer guide](https://docs.dapr.io/contributing/docs-contrib/maintainer-guide/).
+
+  The project does not currently provide information on tag strategies. [Issue #8188 in the Dapr repo](https://github.com/dapr/dapr/issues/8188) has been created to track adding tag strategies to the Dapr process documentation. 
```

Comment:
Is there information sufficient to mark this as met? Our criteria is listed as an "and", however the issue opened may be sufficient to mark this resolved.

Reply:
Updated with latest commit.